tests/common.rs

```diff
@@ -76,3 +77,50 @@ pub async fn start_test_server(
     notify.notified().await;
     h
 }
+
+pub fn array(arr: Vec<&str>) -> String {
+    QueryIO::Array(arr.iter()
+        .map(|s| QueryIO::BulkString(s.to_string()))
+        .collect())
+        .serialize()
+}
+
+pub fn bulk_string(s: &str) -> String {
+    QueryIO::BulkString(s.to_string()).serialize()
+}
+
+pub fn ok_response() -> String {
+    QueryIO::SimpleString("OK".to_string()).serialize()
+}
+
+pub fn null_response() -> String {
+    QueryIO::Null.serialize()
+}
+
+pub fn keys_command() -> Vec<u8> {
```

Comment:
how will we allow for keys with pattern?

Reply:
true, made a change for this 💪